@kiva/kv-components/vue/KvCommentsAdd.vue

```diff
@@ -87,14 +94,21 @@ export default {
 
 		const comment = () => {
 			emit(ADD_COMMENT_EVENT, addCommentValue.value);
+			if (props.userMention) {
+				emit(HIDE_INPUT_EVENT);
```

Comment:
Why do we only hide if `userMention`? Is this prop incorrectly named?

Reply:
I would probably just rename this prop to something like `isReply`. "User mention" has different potential meaning, for example tagging a user.